tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "module": "commonjs",
     "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
This was done to prevent scanning of node_modules by tsc. It was returning an error in some node_module.

Reply:
Hmm, I feel like we need this, otherwise we will be missing types like `Electron`. Let me take a look at this.